### PR TITLE
README polishing

### PR DESCRIPTION
**Pull request by aquarat**
I'm familiarising myself with the latest updates and while doing so I polished a few things.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,29 +98,30 @@ docker run -it --rm ^
...
-In order to run Airkeeper locally, you will need to follow these steps:
+Follow these steps to run Airkeeper locally:
 
 1. Open a new terminal and start a new local ethereum node.
 1. Deploy all required contracts (RrpBeaconServer, DapiServer, etc) and set everything up (whitelisting, sponsorship,
    etc).
 1. Switch to the Airkeeper root directory and run `yarn install`.
-1. Add proper values to the `config.json` and `airkeeper.env` files.
-1. Finally run `yarn sls:invoke-local:psp` to invoke the `psp.beaconUpdate` handler function or run
+1. Add appropriate values to the `config.json` and `airkeeper.env` files.
```
Comment (vponline) [inline README.md:117]:
```suggestion
1. Add appropriate values to the `config.json` and `airkeeper.env` files.
```
Shouldn't this be `airkeeper.json` :detective:

**Review by vponline — APPROVED** [review]
Very nice polishing  :smile:

File: README.md (hunk 1 above)
Comment (aquarat) [reply README.md:117]: It is, nicely spotted, I forgot about this PR. Thanks for checking!

**Comment by aquarat** [comment]
I will create a new PR with just the README fixes. Thanks @vponline

**Pull request closed without merging**